app/2.0/docs/tools/polymer-cli.md

```diff
@@ -64,4 +69,61 @@ Polymer CLI works with two types of projects:
 ## Commands
 
 See the documentation on [Polymer CLI commands](polymer-cli-commands) for a list of commands and
-how to use them.
\ No newline at end of file
+how to use them.
+
+## Installing Polymer CLI pre-requisites on Windows 10 {#windows-10}
+
+There are multiple ways to install the pre-requisites for the Polymer CLI on a Windows 10 system. 
+This method uses Bash on the Windows Subsystem for Linux. 
+
+For complete and up-to-date installation instructions for Bash on the Windows Subsystem for Linux, 
+we recommend you review the [Bash on ubuntu on Windows documentation](https://msdn.microsoft.com/en-us/commandline/wsl/about). 
+
+### Enable Bash on Windows 10 {#enable-bash}
+
+1.  Make sure you have an x64 installation of Window 10 with OS Build > 14393.
+    
+1.  Open **Settings** > **Update and Security** > **For developers** and select **Developer Mode**.
+
+1.  From the **Start** Menu, search for **Turn Windows features on or off** and select **Windows 
+Subsystem for Linux (beta)**. 
+
+1.  Restart your computer. 
+
+1.  Log in as Administrator. 
+
+1.  From a PowerShell prompt, run the following command:
+    
+        Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux
+
+1.  Open a Bash prompt:
+    
+        bash
+
+6.  Follow the prompts to create a new user.
+
+Your Bash installation is complete. You can start now start a new Bash prompt by typing `bash` from 
+the Start menu.
+
+### Install Node with nvm {#install-node}
+
+For complete and up-to-date instructions, refer to the [official documentation for installing Node 
+with nvm](https://github.com/creationix/nvm).
+
+1.  Type the following command to use an install script:
+    
+        curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.2/install.sh | bash
```

Comment:
we could also use the master branch - should be always the latest version?
https://raw.githubusercontent.com/creationix/nvm/master/install.sh